src/content/ueber-uns/abteilungen.svx

```diff
@@ -1,57 +1,49 @@
 
-Die Abteilung der Logistik wird von ? organisiert und verwaltet. Sie umfasst insgesamt drei Truppengattungen: Die Instandsetzung, die Versorgung und die Luftlogistik.
+## Einsatz- und Führungsunterstützung
 
-### Pioniertruppe
+### Sanitätsdienst
 
-Die Pioniertruppe setzt sich aus Pionieren und Kampfpionieren zusammen. Die Hauptaufgabe der Pioniere ist die Wartung, Bergung und Instandsetzung eigener Fahrzeuge. Zusätzlich kann es auch zu Spezialanwendungen kommen, wie zum Beispiel der Einsatz eines Brückenlegepanzers durch die Pioniertruppe.
+Die Truppengattung des Sanitätsdienstes wird von Emmi organisiert und verwaltet. Sie befasst sich mit der medizinischen Behandlung, Evakuierung und Endversorgung verwundeter Soldaten. Überwiegend sind Mitglieder des Sanitätsdienstes in die Infanterie eingegliedert, allerdings werden sie auch bei speziellen Unterstützungselementen wie dem beweglichen Arzttrupp (kurz BAT) oder dem MEDEVAC (MEDical EVACuation) eingesetzt. Der [Sanitäter](../sanitaetsdienst/sanitaeter/) ist die vielfältige Rolle des Sanitätsdienstes.
 
-### Versorgung
+### Logistiktruppe
 
-Die Truppengattung der Versorgung befasst sich mit der Versorgung anderer Truppenteile mit Bodenfahrzeugen. Häufig transportiert werden Güter wie Munition, Treibstoff und Verbrauchsmaterialien. Gegebenenfalls kann die Versorgung auch eigene Kräfte ins Feld verlegen.
+Die Truppengattung der Logistiktruppe wird von ? organisiert und verwaltet. Sie befasst sich mit dem Nachschub, dem Transport und der Instandsetzung. Mit Land- und Wasserfahrzeugen stellen sie sicher, dass den Kampftruppen ausreichend Verbrauchsmaterial (Munition, Sanitätsmaterial), Ersatzreifen und -ketten, Treibstoff und Fahrzeugmunition zur Verfügung steht. Darüber hinaus übernimmt die Logistiktruppe die (Nach-) Verlegung von Nachzüglern über den Landweg und sorgt für die Instandsetzung von Fahrzeugen. Die Logistiktruppe umfasst die Rolle des [Logistikers (DEAD)](../logistik/logistiker).
 
-### Luftversorgung
+### Aufklärungstruppe
 
-Die Luftlogistik befasst sich im Kern mit der Versorgung anderer Truppenteile mit Luftfahrzeugen. Transportierte Güter sind hierbei Munition, Treibstoff und Verbrauchsmaterialien. Die Luftversorgung kann sich ebenfalls mit der Verlegung eigener Kräfte ins Feld befassen.
+Die Truppengattung der Aufklärungstruppe wird von ? verwaltet und organisiert. Ihre Hauptaufgabe ist die Feldaufklärung feindlicher Truppenbewegungen während und außerhalb des Gefechts. Darüber hinaus werden Artilleriefeuer und Luftnahunterstützung durch die Aufklärungstruppe koordiniert und organisiert. Die Aufklärungstruppe umfasst die Rollen des [Scharfschützen](../aufklaerer/scharfschuetze), des [Beobachters](../aufklaerer/spotter) und des [FAC](../aufklaerer/jtac).
```

Comment:
```suggestion
Die Truppengattung der Aufklärungstruppe wird von ? verwaltet und organisiert. Ihre Hauptaufgabe ist die Feldaufklärung feindlicher Truppenbewegungen während und außerhalb des Gefechts. Darüber hinaus werden Artilleriefeuer und Luftnahunterstützung durch die Aufklärungstruppe koordiniert und organisiert. Die Aufklärungstruppe umfasst die Rollen des [Scharfschützen](../aufklaerer/scharfschuetze), des [Beobachters](../aufklaerer/spotter) und des [JTAC](../aufklaerer/jtac).
```

Reply:
This was done on purpose. The standard NATO term for this is "Forward Air Controller" (FAC). The term "Joint Terminal Attack Controller" is [primarily being used by the United States Armed Forces](https://en.wikipedia.org/wiki/Forward_air_control#NATO). Since we do not roleplay US Forces, I found it more sensible to use the standard NATO term for this.

@pantoffelemmy @Venrix Feel free to comment your opinion on this.

Reply:
If FAC is nato terminology I'm for it. Let's just also change the JTAC article accordingly.